src/app/components/DataDetailPage.tsx

```diff
@@ -0,0 +1,237 @@
+'use client';
+
+import { Fragment } from 'react';
+import { Box, Flex, Text, Icon, Link, Image } from '@chakra-ui/react';
+import { MdLocationOn } from 'react-icons/md';
+import { RiBuildingFill } from 'react-icons/ri';
+import { AiOutlineDollar, AiOutlineClose } from 'react-icons/ai';
+import { TbBrandGoogleAnalytics } from 'react-icons/tb';
+import styled from 'styled-components';
+
+const Btn = styled.button`
```

Comment:
Need to gray border to the pop-up page from the designs.

<img width="268" alt="image" src="https://user-images.githubusercontent.com/5522386/230511809-0851615f-777c-48b2-a5bc-600348718364.png">

Reply:
should look like this:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/5522386/230511949-9f4771a5-9245-4674-83db-c93e3a84762b.png">